httpclient/src/main/java/com/baeldung/ClientTlsVersionExamples

```diff
@@ -0,0 +1,71 @@
+package com.baeldung
```

Comment:
The file should be named as a .java file, and the package should be specific to the article, e.g. com.baeldung.tlsversion

Reply:
Moved to com.baeldung.tlsversion